modules/apps/dynamic-data-mapping/dynamic-data-mapping-web/src/main/java/com/liferay/dynamic/data/mapping/web/internal/info/field/converter/DDMFormFieldInfoFieldConverterImpl.java

```diff
@@ -49,9 +49,24 @@ public InfoField convert(DDMFormField ddmFormField) {
 				label.getDefaultLocale()
 			).build();
 
-		return new InfoField(
+		InfoField infoField = new InfoField(
 			_getInfoFieldType(ddmFormField), labelInfoLocalizedValue,
 			ddmFormField.isLocalizable(), ddmFormField.getName());
+
+		if (Objects.equals(
+				ddmFormField.getType(), DDMFormFieldType.TEXT_HTML)) {
+
+			infoField.addAttribute(TextInfoFieldType.MULTILINE, true);
+			infoField.addAttribute(TextInfoFieldType.RICH, true);
```

Comment:
It might not be important, but I wonder if "Rich" could not be clear enough. Maybe "RICH_TEXT" would be better?
I'm even thinking that, since we are doing it at the field level and not at the field type level by using an attribute, it should be "HTML" to be more specific of what it really is.
What do you think?

Reply:
HTML is probably more accurate. I'll change it tomorrow if this one passes or in the next attempt.

Thanks!